### PR TITLE
fix 'oadm policy' description error

### PR DESCRIPTION
**Pull request by miaoyq**
Fix `oadm policy` description error.

Fixes https://github.com/openshift/origin/issues/7195

### CODE REVIEW

File: pkg/cmd/admin/policy/modify_roles.go [hunk 1]
```diff
@@ -103,8 +103,8 @@ func NewCmdRemoveRoleFromGroup(name, fullName string, f *clientcmd.Factory, out
 
 	cmd := &cobra.Command{
 		Use:   name + " ROLE GROUP [GROUP ...]",
-		Short: "Remove group from role in the current project",
-		Long:  `Remove group from role in the current project`,
+		Short: "Remove role from group in the current project",
+		Long:  `Remove role from group in the current project`,
```
Comment (liggitt) [inline pkg/cmd/admin/policy/modify_roles.go]: `a role`

File: pkg/cmd/admin/policy/modify_roles.go [hunk 2]
```diff
@@ -128,8 +128,8 @@ func NewCmdRemoveRoleFromUser(name, fullName string, f *clientcmd.Factory, out i
 
 	cmd := &cobra.Command{
 		Use:   name + " ROLE USER [USER ...]",
-		Short: "Remove user from role in the current project",
-		Long:  `Remove user from role in the current project`,
+		Short: "Remove role from user in the current project",
+		Long:  `Remove role from user in the current project`,
```
Comment (liggitt) [inline pkg/cmd/admin/policy/modify_roles.go]: `a role from users`

File: pkg/cmd/admin/policy/modify_roles.go [hunk 3]
```diff
@@ -197,8 +197,8 @@ func NewCmdRemoveClusterRoleFromGroup(name, fullName string, f *clientcmd.Factor
 
 	cmd := &cobra.Command{
 		Use:   name + " <role> <group> [group]...",
-		Short: "Remove group from role for all projects in the cluster",
-		Long:  `Remove group from role for all projects in the cluster`,
+		Short: "Remove role from group for all projects in the cluster",
+		Long:  `Remove role from group for all projects in the cluster`,
```
Comment (liggitt) [inline pkg/cmd/admin/policy/modify_roles.go]: `a role from groups`

File: pkg/cmd/admin/policy/modify_roles.go [hunk 4]
```diff
@@ -219,8 +219,8 @@ func NewCmdRemoveClusterRoleFromUser(name, fullName string, f *clientcmd.Factory
 
 	cmd := &cobra.Command{
 		Use:   name + " <role> <user> [user]...",
-		Short: "Remove user from role for all projects in the cluster",
-		Long:  `Remove user from role for all projects in the cluster`,
+		Short: "Remove role  from user for all projects in the cluster",
```
Comment (liggitt) [inline pkg/cmd/admin/policy/modify_roles.go]: remove double space, `a role from users`

**Comment by miaoyq** [comment]
I have updated the codes, PTAL @liggitt

File: pkg/cmd/admin/policy/modify_roles.go [hunk 5]
```diff
@@ -128,8 +128,8 @@ func NewCmdRemoveRoleFromUser(name, fullName string, f *clientcmd.Factory, out i
 
 	cmd := &cobra.Command{
 		Use:   name + " ROLE USER [USER ...]",
-		Short: "Remove user from role in the current project",
-		Long:  `Remove user from role in the current project`,
+		Short: "Remove a role from user in the current project",
+		Long:  `Remove a role from user in the current project`,
```
Comment (liggitt) [inline pkg/cmd/admin/policy/modify_roles.go]: `Remove a role from users for the current project`

File: pkg/cmd/admin/policy/modify_roles.go [hunk 6]
```diff
@@ -52,8 +52,8 @@ func NewCmdAddRoleToGroup(name, fullName string, f *clientcmd.Factory, out io.Wr
 
 	cmd := &cobra.Command{
 		Use:   name + " ROLE GROUP [GROUP ...]",
-		Short: "Add groups to a role in the current project",
-		Long:  `Add groups to a role in the current project`,
+		Short: "Add a role to groups in the current project",
+		Long:  `Add a role to groups in the current project`,
```
Comment (liggitt) [inline pkg/cmd/admin/policy/modify_roles.go]: let's change "in the current project" to "for the current project"

File: pkg/cmd/admin/policy/modify_roles.go [hunk 7]
```diff
@@ -103,8 +103,8 @@ func NewCmdRemoveRoleFromGroup(name, fullName string, f *clientcmd.Factory, out
 
 	cmd := &cobra.Command{
 		Use:   name + " ROLE GROUP [GROUP ...]",
-		Short: "Remove group from role in the current project",
-		Long:  `Remove group from role in the current project`,
+		Short: "Remove a role from group in the current project",
+		Long:  `Remove a role from group in the current project`,
```
Comment (liggitt) [inline pkg/cmd/admin/policy/modify_roles.go]: "groups for the current project"

File: pkg/cmd/admin/policy/modify_scc.go [hunk 8]
```diff
@@ -44,8 +44,8 @@ func NewCmdAddSCCToGroup(name, fullName string, f *clientcmd.Factory, out io.Wri
 
 	cmd := &cobra.Command{
 		Use:   name + " SCC GROUP [GROUP ...]",
```
Comment (liggitt) [inline pkg/cmd/admin/policy/modify_scc.go]: undo these changes... need to think about how to phrase this better... the data model really does add users/groups to SCCs

**Comment by miaoyq** [comment]
I have updated `modify_roles.go`, and undone changes of `modify_scc.go`. But we may need to change the order of the "SCC".  @liggitt

File: pkg/cmd/admin/policy/modify_roles.go [hunk 9]
```diff
@@ -103,8 +103,8 @@ func NewCmdRemoveRoleFromGroup(name, fullName string, f *clientcmd.Factory, out
 
 	cmd := &cobra.Command{
 		Use:   name + " ROLE GROUP [GROUP ...]",
-		Short: "Remove group from role in the current project",
-		Long:  `Remove group from role in the current project`,
+		Short: "Remove a role from group for the current project",
+		Long:  `Remove a role from group for the current project`,
```
Comment (liggitt) [inline pkg/cmd/admin/policy/modify_roles.go]: groups

File: pkg/cmd/admin/policy/modify_roles.go [hunk 10]
```diff
@@ -128,8 +128,8 @@ func NewCmdRemoveRoleFromUser(name, fullName string, f *clientcmd.Factory, out i
 
 	cmd := &cobra.Command{
 		Use:   name + " ROLE USER [USER ...]",
-		Short: "Remove user from role in the current project",
-		Long:  `Remove user from role in the current project`,
+		Short: "Remove a role from user for the current project",
+		Long:  `Remove a role from user for the current project`,
```
Comment (liggitt) [inline pkg/cmd/admin/policy/modify_roles.go]: users

File: pkg/cmd/admin/policy/modify_roles.go [hunk 11]
```diff
@@ -197,8 +197,8 @@ func NewCmdRemoveClusterRoleFromGroup(name, fullName string, f *clientcmd.Factor
 
 	cmd := &cobra.Command{
 		Use:   name + " <role> <group> [group]...",
-		Short: "Remove group from role for all projects in the cluster",
-		Long:  `Remove group from role for all projects in the cluster`,
+		Short: "Remove a role from group for all projects in the cluster",
+		Long:  `Remove a role from group for all projects in the cluster`,
```
Comment (liggitt) [inline pkg/cmd/admin/policy/modify_roles.go]: groups

File: pkg/cmd/admin/policy/modify_roles.go [hunk 12]
```diff
@@ -219,8 +219,8 @@ func NewCmdRemoveClusterRoleFromUser(name, fullName string, f *clientcmd.Factory
 
 	cmd := &cobra.Command{
 		Use:   name + " <role> <user> [user]...",
-		Short: "Remove user from role for all projects in the cluster",
-		Long:  `Remove user from role for all projects in the cluster`,
+		Short: "Remove a role from user for all projects in the cluster",
+		Long:  `Remove a role from user for all projects in the cluster`,
```
Comment (liggitt) [inline pkg/cmd/admin/policy/modify_roles.go]: users

**Comment by liggitt** [comment]
a few pluralization changes, also need to run `hack/update-generated-docs.sh` to update example doc

**Comment by liggitt** [comment]
[test]

**Comment by miaoyq** [comment]
I'm so sorry, I was careless.  I have run `hack/update-generated-docs.sh`. PTAL Thanks! @liggitt

**Comment by liggitt** [comment]
[test]

**Comment by liggitt** [comment]
@miaoyq it looks like your system clock is off, which means your commits show up as having occurred after the instructions to the bot to test. can you fix your system clock, and re-create/re-push your commit?

**Comment by deads2k** [comment]
https://github.com/openshift/origin/pull/10606 merged. closing this.
